### PR TITLE
Step 6: Reference GitHub repository using SSH instead of HTTPS

### PR DESCRIPTION
**Pull request by emadbaqeri**
Due to the privacy update on GitHub, the normal sign-in with username and password feature has been deprecated. By following step six and adding HTTP remote as an upstream branch for deploying the app by `gh-pages`, the process is going to be terminated and you'll see an error from GitHub.

### CODE REVIEW

**Review by Ooverapp77 — APPROVED** [review]

**Comment by gitname** [comment]
Hi @emadbaqeri, sorry I did not provide an update on this PR in several months.

I think there are pros and cons to using SSH vs HTTPS (e.g. SSH requires generating a key pair; HTTPS requires generating a personal access token; the SSH port is blocked by some corporate firewalls; the personal access token is specific to GitHub whereas an SSH key pair _can_ be reused across servers).

The change I plan to make to the tutorial to address the underlying issue (i.e. the inability to push to the remote repo) is to refer the reader to GitHub's documentation about accessing remote repositories via HTTPS URLs ([here](https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls)) and SSH URLs ([here](https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-ssh-urls)). I will leave it to the reader to get himself/herself to the point of being able to push to remote repos.

Thanks again for bringing the issue to my attention. I would like to credit you for doing so. Are you OK with me adding your GitHub avatar to [the _Contributors_ section](https://github.com/gitname/react-gh-pages#contributors) at the bottom of the README? This is optional (I will assume "no" unless I hear otherwise from you).

**Comment by emadbaqeri** [comment]
> Hi @emadbaqeri, sorry I did not provide an update on this PR in several months.
> 
> I think there are pros and cons to using SSH vs HTTPS (e.g. SSH requires generating a key pair; HTTPS requires generating a personal access token; the SSH port is blocked by some corporate firewalls; the personal access token is specific to GitHub whereas an SSH key pair _can_ be reused across servers).
> 
> The change I plan to make to the tutorial to address the underlying issue (i.e. the inability to push to the remote repo) is to refer the reader to GitHub's documentation about accessing remote repositories via HTTPS URLs ([here](https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls)) and SSH URLs ([here](https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-ssh-urls)). I will leave it to the reader to get himself/herself to the point of being able to push to remote repos.
> 
> Thanks again for bringing the issue to my attention. I would like to credit you for doing so. Are you OK with me adding your GitHub avatar to [the _Contributors_ section](https://github.com/gitname/react-gh-pages#contributors) at the bottom of the README? This is optional (I will assume "no" unless I hear otherwise from you).

Hey there, no worries at all. Thanks for your updating me about the outcome and your idea sounds cool actually. Yeah sure fill free to do so, that would be my honor actually 🤗

**Pull request closed without merging**
